DontPanicLabs.CodeAnalysis/Sample.cs

```diff
@@ -0,0 +1,126 @@
+// Licensed to the.NET Foundation under one or more agreements.
+// The.NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace DontPanicLabs.CodeAnalysis
+{
+    public class Sample
+    {
+        public const int PublicConstant = 42;
+        private const int PrivateConstant = 42;
+
+        public static readonly int PublicStaticReadonlyField = 42;
+        internal static readonly int s_internalStaticReadonlyField = 42;
+        private static readonly int s_privateStaticReadonlyField = 42;
+
+        internal static int s_internalStaticField = 42;
+        private static int s_privateStaticField = 42;
+
+        private readonly int _privateReadonlyField = 42;
+
+        internal int _internalField = 42;
```

Comment:
Personally, I'd disable any non-private fields, but the editor config rule only seems to work on public.